.github/workflows/sync_docs.md

```diff
@@ -21,3 +21,35 @@ jobs:
       contents: write  # Needed to push branch & tag
       pull-requests: write  # Needed to create PR
 ```
+
+## Documentation requirements
+
+Requirement: The overview topic contains a navigation table with the columns "Level", "Path", and "Navlink"
+
+Requirement: The navigation table of the overview page is wrapped by the `[details]` HTML/markdown element as follows:
+
+```
+# Navigation
+
+[details=Navigation]
+
+<navigation table goes here>
+
+[/details]
+
+```
+
+> [!NOTE]  
+> There should be a white space before and after the `[details]` and `[/details]` lines. Otherwise, Discourse/Charmhub may not parse them correctly.
+
+Requirement: Links in the `Navlink` column are formatted as `[Text](/t/<number>)`. 
+
+They are **not** formatted as `[Text](/t/<some-additional-slug-text>/<number>)`.
```

Comment:
I was using the present tense because it's a syntax I've seen a lot in requirements, but I don't really like it either.

Will change req wording to "must" and "must not" to make them more straightforward.